docs/src/modules/java/pages/sanitization.adoc

```diff
@@ -0,0 +1,106 @@
+= Sanitization
```

Comment:
Shall we also link to this page from one of the agent pages since that might be the most common usage? From the guardrails page?